backend/public/businessApiResponse.json

```diff
@@ -0,0 +1,1452 @@
+{
+  "html_attributions": [],
+  "next_page_token": "AUc7tXUP00hIca5LrgGWxzamPV9RRGTKYQIfRndtovTCtS7iJcR3tV8jFSfzgH8_zXlxpcR9GZgIcdigVADeocA85sfwwIB_cq2GbteANraUiNoU5UEMMCdIKYGztxzugyrljmmkr3SUolpPZXYc3ud7u9ikuNrXrK9eE89zRVgpnNJXqp47DSYTFrz03nsP0FfCWFaiF7s4MMKGASGAlz4oKIEfQaJidInp8tkvJ9625If84fL1Rk0dLE32dWCRo4z_BPZ2sH39eroxHLEwxbuMHvXj8ty9zyb1K1682MboTgVr3eai87uydzxsYw3WQAdhWZlLTbElMbw6UkY52migsC0k9PqrLIeRWl2flfJxj4uJiTMPrtvVZXAun8ploOw0Lx3kTNMkZE1m3gL2Fm8j_ZszoLwlzHYwlbnbTz72gCgLx9_9nXfDQ9by6gbv",
+  "results": {
+    "Food and Dining": [
```

Comment:
Grouping businesses by category is something we should be doing in the logic of your application, not in the data itself.

Let's consider this json file as a placeholder, but we should follow the schema you created in your project plan more closely.

```
ID (Primary Key)
Name
Location
Contact Information
Business Hours
Services Offered
Ratings and Reviews (average)
Business type/category
```